src/pages/TrafficList/TrafficListComponent.tsx

```diff
@@ -0,0 +1,335 @@
+import * as FilterComponent from '../../components/FilterList/FilterComponent';
+import { ThresholdStatus, NA } from 'types/Health';
+import { NodeType, hasProtocolTraffic, ProtocolTraffic } from 'types/Graph';
+import { getTrafficHealth } from 'types/ErrorRate';
+import history, { URLParam } from 'app/History';
+import { createIcon } from 'components/Health/Helper';
+
+export interface TrafficListItem {
+  healthStatus: ThresholdStatus;
+  icon: string;
+  node: TrafficNode;
+  protocol: string;
+  trafficRate: string;
+  trafficPercent: string; // percent success
+}
+
+type TrafficListComponentProps = FilterComponent.Props<TrafficListItem> & {
+  direction: 'inbound' | 'outbound';
+  trafficItems: TrafficItem[];
+};
+
+type TrafficListComponentState = FilterComponent.State<TrafficListItem> & {
+  sortBy: ISortBy;
+};
+
+const columns = [
+  {
+    title: 'Name',
+    transforms: [sortable]
+  },
+  {
+    title: 'Protocol',
+    transforms: [sortable]
+  },
+  {
+    title: 'Rate',
+    transforms: [sortable]
+  },
+  {
+    title: 'Percent',
+    transforms: [sortable, cellWidth(5) as any]
+  },
+  {
+    title: 'TrafficStatus',
+    transforms: [sortable]
+  },
+  {
+    title: 'Actions'
+  }
+];
+
+// Style constants
+const containerPadding = style({ padding: '20px 20px 20px 20px' });
+
+class TrafficListComponent extends React.Component<TrafficListComponentProps, TrafficListComponentState> {
+  constructor(props: TrafficListComponentProps) {
+    super(props);
+    this.state = {
+      currentSortField: this.props.currentSortField,
+      isSortAscending: this.props.isSortAscending,
+      listItems: [],
+      sortBy: {}
+    };
+  }
+
+  // It invokes backend when component is mounted
+  componentDidMount() {
+    this.updateListItems();
+  }
+
+  componentDidUpdate(prevProps: TrafficListComponentProps, _prevState: TrafficListComponentState, _snapshot: any) {
+    const paramChange =
+      prevProps.trafficItems !== this.props.trafficItems ||
+      prevProps.isSortAscending !== this.props.isSortAscending ||
+      prevProps.currentSortField.title !== this.props.currentSortField.title;
+
+    if (paramChange) {
+      this.setState({
+        currentSortField: this.props.currentSortField,
+        isSortAscending: this.props.isSortAscending
+      });
+
+      this.updateListItems();
+    }
+  }
+
+  render() {
+    return (
+      <div className={containerPadding}>
+        <Table
+          aria-label="Sortable Table"
+          sortBy={this.state.sortBy}
+          cells={columns}
+          rows={this.rows()}
+          onSort={this.onSort}
+        >
+          <TableHeader />
+          {this.state.listItems.length > 0 ? (
+            <TableBody />
+          ) : (
+            <tr>
+              <td colSpan={columns.length}>
+                <EmptyState variant={EmptyStateVariant.full}>
+                  <Title headingLevel="h5" size="lg">
+                    `No {this.props.direction} Traffic`
```

Comment:
No need to add the `` quotes
![image](https://user-images.githubusercontent.com/1662329/94266398-61aaf000-ff3a-11ea-84b0-24e811413c55.png)